src/autogluon_dashboard/dashboard.py

```diff
@@ -139,7 +159,7 @@ def run_dashboard():
     logger.info("WebAssembly files have been successfully uploaded to bucket - %s", bucket_name)
 
     logger.info(
-        "The dashboard website is: " + f"https://{bucket_name}.s3-website-{region}.amazonaws.com/{prefix}/out.html"
+        "The dashboard website is: " + f"http://{bucket_name}.s3-website-{region}.amazonaws.com/{prefix}out.html"
```

Comment:
Is the default url given by s3 http? We probably want to change this. Add a todo maybe